unit_tests/src/constants.rs

```diff
@@ -6,5 +6,70 @@ pub const STARKGATE_ETH_CONTRACT_ADDR: &str = "0x049d36570d4e46f48e99674bd3fcc84
 pub const INVALID_CONTRACT_ADDR: &str = "0x4269DEADBEEF";
 pub const CONTRACT_ADDR: &str = "0x03a20d4f7b4229e7c4863dab158b4d076d7f454b893d90a62011882dc4caca2a";
 pub const CONTRACT_KEY: &str = "0x00f920571b9f85bdd92a867cfdc73319d0f8836f0e69e06e4c5566b6203f75cc";
+
+/**
+ * Contract address for StarkGate ETH starknet bridge.
```

Comment:
Same here please use rustdoc official nomination ///

Reply:
Updated all comments in commit cb063efab7adad0d47d778a6ddcfb453dae72e36 and 2a4c95f3d9f6708715b6f8d34d1486ee42eaf2c0